pages/api/toDOH/index.js

```diff
@@ -0,0 +1,39 @@
+import { client } from "../../../utils/Database";
+
+export default async function handler(request, response) {
+  try {
+    await client.connect();
+    const db = client.db("DOH");
+    const collection = db.collection("toDOHs");
+
+    switch (request.method) {
+      case "GET":
+        const toDOHs = await collection.find().toArray();
+        response.status(200).json(toDOHs);
+        break;
+
+      case "POST":
+        const insertResult = await collection.insertOne(request.body);
```

Comment:
This could potentially be dangerous. You're inserting any `request.body` the client sends you into the collection. This data must not necessarily match your required format.

Reply:
the react doc says that by default all values are escaped before rendering. i hope it really is!